src/unix/core.c

```diff
@@ -1340,3 +1341,12 @@ uv_pid_t uv_os_getpid(void) {
 uv_pid_t uv_os_getppid(void) {
   return getppid();
 }
+
+
+int uv_cpumask_size(void) {
+#if !defined(CPU_SETSIZE)
+  return UV__ERR(ENOTSUP);
```

Comment:
This can just be `UV_ENOTSUP`.

Reply:
The implementation in `master` by #1527 is:

```c
#if defined(__linux__) || defined(__FreeBSD__)
  return CPU_SETSIZE;
#else
  return UV_ENOTSUP;
#endif
```

The idea is to return `UV_ENOTSUP` on platforms where affinity is not supported.  Indeed this PR's `uv_thread_setaffinity` is guarded by the same Linux/FreeBSD condition.